src/components/login/Login.js

```diff
@@ -2,77 +2,210 @@ import React from "react";
+      .catch((error) => {
+        try {
+          alert(error.response.data.error)
+        } catch (error) {
+          alert("Servidor indisponível")
+        }
+      })
+    this.setState({ stateSwitch: 0 });
+  }
+
+  login = () => {
+    return (<>
+      <form onSubmit={this.handleSubmit}>
+        <img src="logo_blue.png" alt="UnB" className="logo" />
+        <input type="text" id="username" value={this.state.username} onChange={this.handleChange} className="form-control login-input" placeholder="Nome de Usuário" />
+        <input type="password" id="password" value={this.state.password} onChange={this.handleChange} className="form-control login-input" placeholder="Senha" />
+        <button className="btn btn-block button-next" type="submit">Entrar</button>
+      </form>
+      <div className="reset" onClick={() => { this.setState({ stateSwitch: 1 }) }}>Esqueceu a sua senha?</div>
+    </>);
+  }
+
+  sendCode = () => {
+    return (<>
+      <img src="logo_blue.png" alt="UnB" className="logo" />
+      <div style={{ color: "red", marginTop: "10px" }}>Digite seu email para trocar de senha</div>
+      <input type="email" id="email" value={this.state.email} onChange={this.handleChange} className="form-control login-input" placeholder="Email" />
+      <button className="btn btn-block button-next" onClick={this.handleEmail}>Enviar</button>
+      <div className="reset" onClick={() => { this.setState({ stateSwitch: 0 }) }}>Voltar</div>
+    </>);
+  }
+
+  confirmCode = () => {
+    return (<>
+      <img src="logo_blue.png" alt="UnB" className="logo" />
+      <div style={{ color: "red", marginTop: "10px" }}>Digite o código que recebeu via email. O código expira em 15 minutos</div>
+      <input type="text" id="code" value={this.state.code} onChange={this.handleChange} className="form-control login-input" placeholder="Código" />
+      <button className="btn btn-block button-next" onClick={this.handleCode}>Enviar</button>
+      <div className="reset" onClick={() => { this.setState({ stateSwitch: 0 }) }}>Voltar</div>
+    </>);
+  }
+
+  loading = () => {
+    return (<>
+      <img src="loading.gif" alt="loading1" />
+    </>);
+  }
 
+  changePassword = () => {
+    return (<>
+      <img src="logo_blue.png" alt="UnB" className="logo" />
+      <div style={{ color: "red", marginTop: "10px" }}>Escolha sua nova senha</div>
+      <input type="password" id="newPassword" value={this.state.newPassword} onChange={this.handleChange} className="form-control login-input" placeholder="Nova Senha" />
+      <input type="password" id="newPasswordConfirm" value={this.state.newPasswordConfirm} onChange={this.handleChange} className="form-control login-input" placeholder="Confirmação da Nova Senha" />
+      <button className="btn btn-block button-next" onClick={this.handleChangePassword}>Enviar</button>
+      <div className="reset" onClick={() => { this.setState({ stateSwitch: 0 }) }}>Voltar</div>
+    </>);
+  }
+
+  switchRender = () => {
```

Comment:
Codacy found an issue: [Arrow function has a complexity of 5. Maximum allowed is 4.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6397155)